### PR TITLE
Bug Smash Session 3

### LINKED ISSUE #387: Improve tabs layout on mobile
Improve layout of `profilul meu` and `alte persoane` on mobile screens.

Suggestion:
Google Pixel 2 - 1

### LINKED ISSUE #390: My profile details should not be displayed in Completează formularul de evaluare section
The `https://dev.stamacasa.ro/evaluation/me` page should only have the form button and not the profile details.

Nice-to-have: show the form directly and not the yellow button `Completeaza formularul`

### LINKED ISSUE #391: Update footer and header links
There should be the same links in header (+ register & login) & footer as in https://diasporahub.ro/

### LINKED ISSUE #392: Always show alte simptome subtitle in history even if there are no data
In `Istoric Simptome` show the `alte simptome` subtitle even if there are no data. 

Screenshot from 2020-04-28 22-00-45

The correct behavious would be to sheow the `Alte simptome` subtitle with the `Nu exista date` content.

### LINKED ISSUE #393: Add more padding in my profile layout
In `https://dev.stamacasa.ro/account/me`

Add padding between:
- paragraph text `.....Poți completa formularul aici.` and title `Istoric Simptome`
- `Istoric Simptome` data and footer

### PR DESCRIPTION
**Pull request by moonflare**
### Requirements for making a pull request
- update footer & header links (closes #391)
- improve tabs layout on mobile (closes #387, #375)
- show section title even if data is not available (closes #392)
- add footer & tables spacing (closes #393)
- remove account info from evaluation page (closes #390)
- update first-name last-name order for profileDetails

### CODE REVIEW

File: frontend/src/components/Account/common/ProfileHistory/ProfileHistory.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .profile-history-container {
     margin-bottom: 0px;
     .profile-history-text {
-        margin: 44px 0px 0px 0px;
+        margin: 44px 0px;
```
Comment (surdu) [inline frontend/src/components/Account/common/ProfileHistory/ProfileHistory.scss:4]: Do we want `margin-bottom: 44px` here? I remember being a bug in the past to remove it.

Comment (moonflare) [reply frontend/src/components/Account/common/ProfileHistory/ProfileHistory.scss:4]: Yes, we do. At least, according to https://github.com/code4romania/stam-acasa/issues/393

Comment (surdu) [reply frontend/src/components/Account/common/ProfileHistory/ProfileHistory.scss:4]: Managed to find the original modifications: https://github.com/code4romania/stam-acasa/pull/356/files made for this issue #304

Comment (moonflare) [reply frontend/src/components/Account/common/ProfileHistory/ProfileHistory.scss:4]: Hmm. ok. I can see that issue it's related to mobile spacing. Will check it out later tonight and make sure we only have the new changes applied for desktop.

Comment (moonflare) [reply frontend/src/components/Account/common/ProfileHistory/ProfileHistory.scss:4]: Checked it and looks ok on mobile. The previous styles were for `.profile-history-container`. IMO the spacing is fine here, but thanks for mentioning it 👌 
Screenshot 2020-04-29 at 18 07 51

**Review by aniri — APPROVED** [review]
tested locally and works well :+1:

**Review by surdu — APPROVED** [review]

**Comment by surdu** [comment]
Also fixes #375

**Pull request merged**